ActionEventsLoader.lua

```diff
@@ -323,6 +328,12 @@ function ActionEventsLoaderUtil.getActionEventText(actionEventData)
 	return actionEventData.text or ActionEventsLoader.textPrefix..actionEventData.name
 end
 
+---Needs the action event text to be visible?
+---@param table actionEventData from the config xml
+function ActionEventsLoaderUtil.isActionEventTextVisible(actionEventData)
+	return courseplay.globalSettings.showActionEventsTexts:get() or actionEventData.textVisibilityNeeded
```

Comment:
would `textAlwaysVisible` describe `textVisibilityNeeded` better what it is for?